src/rockstor/storageadmin/static/storageadmin/js/templates/common/navbar.jst

```diff
@@ -40,6 +40,7 @@
       <li><a href="#config-backup"><i class="glyphicon glyphicon-floppy-save"></i> Config Backups</a></li>
       <li><a href="#email"><i class="fa fa-envelope"></i> Email Alerts</a></li>
       <li><a href="#logsmanager"><i class="fa fa-gears"></i> Logs Manager</a></li>
+      <li><a href="#shell"><i class="fa fa-terminal"></i> System Shell</a></li>
```

Comment:
Eheh here comes another problem I had to challenge with: Rockstor webUI available space

I was not sure about shell icon so got a "code" icon to have a different icon (from font awesome, open to suggestions !)
Current system menu is becoming too long (IMO) and you're right we could move shell icon next to system time, but on that bar we have hostname, management info and system time on left, kernel and uptime on right, so i'm not sure if it's better to have shell on system menu or on this bar... @priyaganti @phillxnet ??? :smiley: